src/components/admin/FlowTypes.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+export type UserType = {
+  adminNote: string,
+  email: string,
+  isAdmin: boolean,
+  joinDate: string,
```

Comment:
is there a reason for not using a date type?

Reply:
As long as we don't have to manipulate the date (other than setting it initially that is) or use it in any logic this seems fine for me

In general, keep in mind that time and date is very relative (e.g different time zones), so server time might be differ from the user's time.